README.md

```diff
@@ -178,6 +178,20 @@ await db
 // INSERT INTO users (email, age) VALUES ('a@b.com', 16);
 ```
 
+### Replace data
+
+```ts
+await db
+  .queryBuilder("users")
+  .replace({
+    email: "a@b.com",
+    age: 16,
+    created_at: new Date("5 June, 2020"),
+  })
+  .execute();
+// REPLACE INTO users (email, age) VALUES ('a@b.com', 16);
```

Comment:
I think you forgot to add the `created_at` column to the SQL result.

```sql
REPLACE INTO users (email, age, created_at) VALUES ('a@b.com', 16, '2020-06-05 00:00:00')
```

It's not your fault, I forgot this column in the `insert` example too 😅.